### ISSUE TITLE
Counter - fix accessibility violations

### ISSUE THREAD

**Issue by thrbnhrtmnn** [deven-org/B01LER-Kitchen#776]
#### Description / User Story

The following errors and incomplete tests are thrown by the storybook accessibility addon and should be fixed:
- Incomplete: Elements must have sufficient color contrast (size=md)

----
#### Requirements
- [x] #543 

----
#### Acceptance Criteria
- [ ] All errors and incomplete tests thrown by the accessibility addon are resolved:
   - [x] Incomplete: Elements must have sufficient color contrast (size=md)

----
#### Background information
- ...

**Comment by JpunktWpunkt** [comment]
I have checked the colour values in sm, md and lg and they are all passed correctly
- the accessibility test shows the correct standard for sm and lg
- in md the message remains
- I have checked the contrast ratio manually and the values are WCGA 2 AA compliant
- Sometimes, the Accessibility extension didn't recognize and throws an error, which in actual is not an error. Maybe this is that issue in that case here.

**Comment by thrbnhrtmnn** [comment]
As discussed today, we will close the issue. The only reason that the incomplete error is thrown seems to be because the accessibility add-on is not functioning correctly. The issue is not appearing in smaller or in larger sizes of the component and the colors are also compliant.
